pkg/backingstore/reconciler.go

```diff
@@ -1173,9 +1173,18 @@ func (r *Reconciler) needUpdate(pod *corev1.Pod) bool {
 	var configMapLogLevel = r.CoreAppConfig.Data[noobaaLogEnv]
 	noobaaLogEnvVar := util.GetEnvVariable(&c.Env, noobaaLogEnv)
 
-	if (configMapLogLevel != noobaaLogEnvVar.Value) {
+	if configMapLogLevel != noobaaLogEnvVar.Value {
 		r.Logger.Warnf("NOOBAA_LOG_LEVEL Env variable change detected: (%v) on the config map (%v)", noobaaLogEnvVar.Value, configMapLogLevel)
-			return true
+		return true
+	}
+
+	var noobaaLogColEnv = "NOOBAA_LOG_COLOR"
+	var configMapLogCol = r.CoreAppConfig.Data[noobaaLogColEnv]
+	noobaaLogColEnvVar := util.GetEnvVariable(&c.Env, noobaaLogColEnv)
+
+	if configMapLogCol != noobaaLogColEnvVar.Value {
+		r.Logger.Warnf("NOOBAA_LOG_COLOR Env variable change detected: (%v) on the config map (%v)", noobaaLogColEnvVar.Value, configMapLogCol)
+		return true
 	}
```

Comment:
seems like we can make it the same code, and itirate on the variables names. 
something like:
```for _, name := range []string{"NOOBAA_LOG_LEVEL", "NOOBAA_LOG_COLOR"} {```

Reply:
Done.